### DISCUSSION TITLE
Different spaCy label for local output than on the explosion.ai website

### DISCUSSION THREAD

**Discussion by defreeze** [explosion/spaCy#12056] [Language Support]
The website of explosion.ai has a section to add text and then it uses displacy to show the labeling and relations: 
https://demos.explosion.ai/displacy?text=Wie%20is%20Abraham%3F&model=nl_core_news_sm&cpu=1&cph=1

When i use the exact same code, i get different results for the Dutch language then shown on the website.
I need the results shown on the website, as those are correct. It is required, otherwise i cannot implement spaCy for the Dutch language.

See the attached picture (left, local code in visual studio code. Right the output on explosion.ai website)
<img width="948" alt="spacy output website and local" src="https://user-images.githubusercontent.com/17338063/210530074-2509b928-dea1-4175-b9fb-384b82738e60.png">
'is' should be an AUX not a VERB.

Spacy version is 2.3.9. I get te same results with each version of the Dutch model (small medium and large).

**Comment by thomashacker** [comment]
Hello,

If you're using spaCy v2.3.9, you're probably using an older version of the dutch model. The displaCy demo uses the newest version of the model v3.4.0, which might explain the different outputs.

**Reply by defreeze** [reply]
Thanks Thomas,

This is very likely it! It auto installs -> nl_core_news_lg==2.3.0.

Is there a way to use the latest version (v3.4.0), without changing spaCy version?
On the website it is possible to manualy download that version as whl file (https://spacy.io/models/nl).
Thanks for the quick response.

Alex

**Reply by thomashacker** [reply]
Unfortunately, you can't use v3.x models with spaCy v2.x

**Reply by defreeze** [reply]
Thanks a lot for the response Thomas.
Indeed I tested it and this is correct. If I install spaCy v3.x it will download nl-core-news-lg-3.4.0. This then gives me the correct results!

One last question is that I need to use neuralcoref, which seems to only work with spaCy v2.x. Is this still the case? Installing it from source still gives errors. I'll update this threat if I find a solution, but any tips are appreciated.

**Reply by thomashacker** [reply]
Yes, you're correct; Neuralcoref only works for spaCy v2.x. 
However, we recently published a new experimental [Coreference component](https://spacy.io/api/coref) with an [example project](https://github.com/explosion/projects/tree/v3/experimental/coref), which showcases how to train and use the component.

We also provide a pretrained coref model (but only for English at the moment), you can download the model from the latest [spacy-experimental release](https://github.com/explosion/spacy-experimental/releases/tag/v0.6.1) and install it via pip.

If you decide you'd like to try out the coref component and need some guidance, feel free to ask in the discussion forums.
For more information we also have a [blogpost](https://explosion.ai/blog/coref) and a [small video](https://www.youtube.com/watch?v=fio3BejnRsM&t=2s&ab_channel=Explosion) about coref.
